src/holofuel/pages/Inbox/Inbox.test.js

```diff
@@ -5,17 +5,16 @@ import { ApolloProvider } from '@apollo/react-hooks'
 import { MockedProvider } from '@apollo/react-testing'
 import moment from 'moment'
 import apolloClient from 'apolloClient'
-import Inbox, { TransactionRow, RenderNickname } from './Inbox'
+import Inbox, { TransactionRow } from './Inbox'
```

Comment:
in this file there is one test (on line 40) that the nicknames are rendered, but it's currently commented out. Can you uncomment that part and make sure the tests are testing that nicknames are rendering